synergy-design-system/synergy-design-system
# feat: ✨ progress ring

# Pull Request

## 📖 Description

This PR adds the new `<syn-progress-ring />` element.

### 🎫 Issues

Closes #381

## 👩‍💻 Reviewer Notes

Just have a look at the provided Storybook documentation.

## 📑 Test Plan

- Check out this branch and build world.
- Spin up storybook
- Have a look at the new Components -> syn-progress-ring stories

## ✅ DoD
